.github/workflows/test-python.yaml

```diff
@@ -12,3 +12,5 @@ jobs:
           python-version: '3.x'
       - run: pip install -r test-requirements.txt
       - run: pytest .
+      - run: python -m pylint ldraw_to_scad.py
```

Comment:
We wouldn't just do *.py? Or are the tests a bit noisy now?

Reply:
The test cases are still as noisy as the code used to be before. I fixed the main code with my restructuring change. I still have the plan to go through the test code though and would add this then once it is in a good shape.

Reply:
Ok - merging this for now.
By the way - perhaps you can check through the issues - I think you may have fixed a few of them with your changes.